wagtail_transfer/locators.py

```diff
@@ -95,7 +106,7 @@ def __init__(self, model, fields):
         self.model = model
         self.fields = fields
 
-    def get_uid_for_local_id(self, id):
```

Comment:
Looks like the API of this defers with `IDMappingLocator.get_uid_for_local_id`

Maybe we should split them into separate `get_uid_for_local_id` and `get_or_create_uid_for_local_id`?

Reply:
Hmm, yes. I thought about that - done now.